### ISSUE TITLE
nu Setup Wizard ended prematurely

### ISSUE THREAD

**Issue by lomirus** [nushell/nushell#11007]
### Describe the bug

Failed to install and it shows the message: `nu Setup Wizard ended prematurely`

### How to reproduce

1. Run the `nu-0.86.0-aarch64-windows-msvc-full.msi`.
2. Just click the Next, keeping all the options as default.
3. Then it failed and showed the message.

### Expected behavior

Successfully installed

### Screenshots

![QQ截图20231109110931](https://github.com/nushell/nushell/assets/21173956/347135fd-b6c3-4467-b9b4-c4cfb24a4b9d)

### Configuration

N/A

### Additional context

OS: Microsoft Windows 11 22621

**Comment by fdncred** [comment]
You might try logging the msi installation and see what the error says.

https://community.flexera.com/t5/InstallShield-Knowledge-Base/Logging-an-MSI-Installation/ta-p/4323

**Comment by lomirus** [comment]
```
msiexec /i "nu-0.86.0-aarch64-windows-msvc-full.msi" /l*v "d:\logs\package.log"
```
Output: [package.log](https://github.com/nushell/nushell/files/13308077/package.log)

**Comment by fdncred** [comment]
MSI (s) (D4:98) [20:28:52:644]: Hello, I'm your 32bit Elevated Non-remapped custom action server.
CAQuietExec:  Error 0x800700d8: Command failed to execute.
CAQuietExec:  Error 0x800700d8: QuietExec Failed
CAQuietExec:  Error 0x800700d8: Failed in ExecCommon method
CustomAction ReplacePathsInWindowsTerminalProfile returned actual error code 1603 (note this may not be 100% accurate if translation happened inside sandbox)
Action ended 20:28:52: InstallFinalize. Return value 3.


These type of errors are pretty hard to diagnose.

**Comment by lomirus** [comment]
Oops, sorry, I just realized I downloaded the wrong architecture. Now fixed by installing `nu-0.86.0-x86_64-windows-msvc-full.msi`😅

**Comment by fdncred** [comment]
lol
